### ISSUE TITLE
Requests logged twice for service

### ISSUE THREAD

**Issue by matthew-white** [getodk/central#265]
I have noticed on a couple of different servers that requests seem to be logged twice for `service`. Interestingly, the requests don't seem to be logged twice for `nginx`. Requests do seem to be logged twice for `service`; I am less sure whether errors and warnings are logged twice, but I think that that's also possible. Even though requests are logged twice, I doubt that two requests are actually being sent, because that would almost certainly lead to user-facing errors that we would have noticed.

**Comment by lognaturel** [comment]
@alxndrsn and I chatted briefly about this and he asked for confirmation that it's always happening and that there's no funny business based on request type or anything like that. From what I can tell, every request is always double logged and it looks like a configuration issue of some sort. 

`docker-compose logs --tail 12 service` from the QA server:
```
service_1 | <ip> - - [24/Jun/2022:18:57:14 +0000] "POST /v1/sessions HTTP/1.0" 401 80
service_1 | <ip> - - [24/Jun/2022:18:57:14 +0000] "POST /v1/sessions HTTP/1.0" 401 80

service_1 | <ip> - - [24/Jun/2022:18:59:14 +0000] "POST /v1/sessions HTTP/1.0" 401 80
service_1 | <ip> - - [24/Jun/2022:18:59:14 +0000] "POST /v1/sessions HTTP/1.0" 401 80

service_1 | <ip> - - [24/Jun/2022:19:00:04 +0000] "GET /v1/test/<key>/projects/149/forms/regex/draft/formList HTTP/1.0" 404 458
service_1 | <ip> - - [24/Jun/2022:19:00:04 +0000] "GET /v1/test/<key>/projects/149/forms/regex/draft/formList HTTP/1.0" 404 458

service_1 | <ip> - - [24/Jun/2022:19:00:04 +0000] "GET /v1/test/<key>/projects/149/forms/random-no-dupes/draft/formList HTTP/1.0" 200 465
service_1 | <ip> - - [24/Jun/2022:19:00:04 +0000] "GET /v1/test/<key>/projects/149/forms/random-no-dupes/draft/formList HTTP/1.0" 200 465

service_1 | <ip> - - [24/Jun/2022:19:00:04 +0000] "GET /v1/test/<key>/projects/149/forms/filteringgameRPG/draft/formList HTTP/1.0" 200 468
service_1 | <ip> - - [24/Jun/2022:19:00:04 +0000] "GET /v1/test/<key>/projects/149/forms/filteringgameRPG/draft/formList HTTP/1.0" 200 468

service_1 | <ip> - - [24/Jun/2022:19:01:14 +0000] "POST /v1/sessions HTTP/1.0" 401 80
service_1 | <ip> - - [24/Jun/2022:19:01:14 +0000] "POST /v1/sessions HTTP/1.0" 401 80
```

Indeed, those requests are logged once for `nginx`.

**Comment by matthew-white** [comment]
Same here, I don't use Docker in local development and gravitate toward a cloud-based environment like the QA server or the dev server when I'm working on ops issues.

**Comment by alxndrsn** [comment]
To reproduce locally from the `central` repository:
```
git submodule update -i
```
In `docker-compose.yml`, replace `driver: local` with `driver: json-file`.

```
docker-compose build service
docker-compose create service
docker-compose start service
docker-compose logs --tail all service
```
Check `central-backend` started ok.
```
curl "$(docker inspect -f "{{.NetworkSettings.Networks.central_default.IPAddress}}" "$(docker ps | grep central_service | cut -d' ' -f1)")":8383
docker-compose logs --tail all service
```

**Comment by alxndrsn** [comment]
Smaller reproduction at https://github.com/alxndrsn/odk-central/tree/double-logging-minimal-repro.

The problem seems to be caused in `pm2.config.js`, by the line:

```
    out_file: '/proc/1/fd/1',
```

Removing that will stop duplicate log entries.  I'm not sure _why_ yet :shrug:

**Comment by alxndrsn** [comment]
The problem exists for output to stderr too.  This can be fixed similarly, by removing the line:
```
    error_file: '/proc/1/fd/2',
```

**Comment by matthew-white** [comment]
Closed by #293.
